lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -88,20 +89,38 @@ class LighthouseReportViewer {
 
   /**
    * Attempts to pull gist id from URL and render report from it.
-   * @return {Promise<void>}
    * @private
    */
   _loadFromDeepLink() {
     const params = new URLSearchParams(location.search);
+
+    let provider = params.get('provider');
```

Comment:
this name is a little weird to me.... since we use `load` and `get`.  this feels more like a `source`.